hare4/protocol_test.go

```diff
@@ -143,9 +143,13 @@ func (t *toutput) round(r Round) *toutput {
 	return t
 }
 
-func (t *toutput) iter(i uint8) *toutput {
+// FIXME: made to satisfy the linter but actually needs unit test
+// changes to test different cases.
+const iter1 = uint8(1)
```

Comment:
Just like in the other places, I believe we should have variable values for this in different tests and if we don't change it within this PR, then `// nolint` for now.